### ISSUE TITLE
Add gruvbox

### ISSUE THREAD

**Issue by circleous** [joonro/ConEmu-Color-Themes#11]
For some weird reason, i cant get gruvbox light working nicely on ConEmu. but here is my dark version.
Color scheme - https://github.com/morhetz/gruvbox-contrib/ 
https://gist.github.com/circleous/92c74d284db392a950d64a2b368517a1

**Comment by joonro** [comment]
Hi - thank you very much for this. I'm very busy til the end of March, but I will try to incorporate this as soon as I have a chance.

**Comment by joonro** [comment]
Hi - I added `gruvbox-dark`. I started with your scheme, and I look for the original theme at https://github.com/morhetz/gruvbox. It was strange, many colors on the palette were slightly different from those in the screenshot. Screenshot looked better, so I used colors in the screenshot. So the resulting color scheme is different from what you originally specified.

Also, I noticed that you had color options (such as `Text`, `Back`, etc) not set `Auto`, which caused foreground colors to be off. (I wonder if that was a reason why you did not get the light color working) I changed that as well, and added a note to the README. Also, in general, I follow the color order defined in the Template at the end of README.

Thanks again!
